crates/sui-graphql-rpc/src/extensions/query_limits_checker.rs

```diff
@@ -106,7 +114,8 @@ impl Extension for QueryLimitsChecker {
             resp.extension(
                 "usage",
                 value! ({
-                    "nodes": validation_result.num_nodes,
+                    "inputNodes": validation_result.num_nodes,
```

Comment:
nit: to rename `num_nodes` to `input_nodes` as well.

Reply:
P.S. thanks for making these camelCase ;)

Reply:
ah right